app/views/users/new.html.erb

```diff
@@ -1,27 +1,41 @@
-<%= form_tag "/users", method: :post do%>
-
-    <%= label_tag :name %>
-    <%= text_field_tag :name %>
-
-    <%= label_tag :address %>
-    <%= text_field_tag :address %>
-
-    <%= label_tag :city %>
-    <%= text_field_tag :city %>
-
-    <%= label_tag :state %>
-    <%= text_field_tag :state %>
-
-    <%= label_tag :zipcode %>
-    <%= text_field_tag :zip %>
-
-    <%= label_tag :email %>
-    <%= text_field_tag :email %>
-
-    <%= label_tag :password %>
-    <%= password_field_tag :password %>
-
-    <%= submit_tag "Create Account"%>
-
-<%end %>
-
+<form action="/register" method=post
+      oninput='password2.setCustomValidity(password2.value != password.value ? "Passwords do not match." : "")'>
+  <div class="form-group">
+    <label for="name">Name</label>
+    <input type="text" class="form-control" id="name" name="name">
+  </div>
+  <div class="form-group">
+    <label for="address">Address</label>
+    <input type="text" class="form-control" id="address" name="address">
```

Comment:
There will be a merge conflict with my code for US12 since I used the old form from Josh's US10.
We will have to discuss this before the PR to master.